resource.address/spi/src/main/java/org/kaazing/gateway/resource/address/ResourceFactories.java

```diff
@@ -17,6 +17,8 @@
 
 import static java.lang.String.format;
 
+import org.kaazing.gateway.resource.address.uri.URIUtils;
```

Comment:
Hello, Jitu, there are some static URIUtils methods used in this class. Therefore the import. The change in the file occurs due to a package refactoring.